srcpkgs/monero-gui/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'monero-gui'
 pkgname=monero-gui
-version=0.17.2.3
-revision=2
-_monero_version=0.17.2.3
-_randomx_version="1.1.9"
+version=0.18.1.2
+revision=1
+_monero_version=0.18.1.2
```

Comment:
this should always be the same as $version, no?

Reply:
It happens (quite frequently, actually) that GUI gets a point release while CLI doesn't. Makes sense, given the difference in complexity.